test/ct.yaml

```diff
@@ -7,6 +7,7 @@ excluded-charts:
   - common
   - dex-controller # Moved to a different helm repo
   - kubefed
+  - kommander-cert-federation # Unable to test unless cert-manager is also upgraded
```

Comment:
might need to update https://github.com/mesosphere/charts/tree/master/test/helm3 ct files